src/Backend/Core/Engine/Authentication.php

```diff
@@ -313,16 +316,40 @@ public static function loginUser(string $login, string $password): bool
     {
         $db = BackendModel::get('database');
 
-        // fetch the encrypted password
-        $passwordEncrypted = static::getEncryptedPassword($login, $password);
+        // build fallback for migrating devs
+        // todo remove in next major release
+        $encryptedPassword = BackendUsersModel::getEncryptedPassword($login);
```

Comment:
I meant 7.2 or 7.3.
If we now include this fallback it needs to be carried on all the way till fork 6.
There is an implementation for password_hash in the security component from symfony meaning that by switching to password hash can be done in a feature release since the implmentation is something internal. But if we would add this fallback we would need to write a custom implementation for it to carry on our old implementation from fork.

The issue was originaly written for 4.5 in a way that it would be backwards compatible. Since this is a major version we can break that